src/createServer.js

```diff
@@ -1,3 +1,38 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { getErrors } = require('./utils/getErrors');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const [originalText, queryString] = req.url.slice(1).split('?');
+    const targetCase = new URLSearchParams(queryString).get('toCase');
+
+    const errors = getErrors(originalText, targetCase);
+
+    if (errors.length > 0) {
+      res.writeHead(400, { 'Content-Type': 'application/json' });
+      res.end(JSON.stringify({ errors }));
+
+      return;
+    }
```

Comment:
Better to throw these errors from your function, and use here `try/catch` to return them to user in `catch`